### PR TITLE
Add Void Linux (#9361)

### LINKED ISSUE #9361: Add Void Linux icon
### Brand Name

Void Linux

### Website

https://voidlinux.org/

### Popularity Metric

The similarweb rank is 304,148 see https://www.similarweb.com/website/voidlinux.org/

### Official Resources for Icon and Color

https://commons.wikimedia.org/wiki/File:Void_Linux_logo.svg

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by ljnsn**
**Issue:** closes #9361 

**Similarweb rank:** 304,148
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] The SVG `viewbox` is `0 0 24 24`
  - [x] I optimized the icon with SVGO or SVGOMG

### Description

I just discarded the text, since I'm not very proficient with SVG editing and I didn't know how to add the gaps between the text and the logo. Anyways, I think this captures the essence of the logo pretty well.

There are two green tones in the logo, I chose the darker one for the hex value.

Preview:

voidlinux

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13891,6 +13891,11 @@
             "hex": "E60000",
             "source": "https://web.vodafone.com.eg"
         },
+        {
+            "title": "Void Linux",
+            "hex": "478061",
+            "source": "https://commons.wikimedia.org/wiki/File:Void_Linux_logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:13897]:
```suggestion
            "source": "http://alpha.de.repo.voidlinux.org/logos/void-dark2.svg"
```

**Review by adamrusted — DISMISSED** [review]
Happy with the shape of this icon, though I'd probably switch the source away from Wikimedia and to the source _they_ used.

File: _data/simple-icons.json (hunk 1 above)
Comment (ljnsn) [reply _data/simple-icons.json:13897]: I was on vacation, thanks for the review and the change makes sense to me!

**Comment by ljnsn** [comment]
@adamrusted would you like to see further changes?

**Review by PratyushJayachandran — APPROVED** [review]
Always good to add the preview with the color to be used
voidlinux

Thank you for contributing @ljnsn and for finsing the source @adamrusted 🎉

**Pull request merged**